service/src/main/java/com/github/szsalyi/flashcard/service/mapper/common/Mapper.java

```diff
@@ -0,0 +1,29 @@
+package com.github.szsalyi.flashcard.service.mapper.common;
+
+
+import org.dozer.DozerBeanMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Mapper {
```

Comment:
While it is an excellent idea to centralize the creation and configuration of a `DozerBeanMapper` instance, let's do it the Spring way. Create a Spring bean from the `DozerBeanMapper` instance so you can inject it as a dependency into other classes.